### PR TITLE
bot testing: Update outdated test for yoda bot.

### PR DESCRIPTION
**Pull request by derAnfaenger**

### CODE REVIEW

File: zulip_bots/zulip_bots/bots/yoda/test_yoda.py [hunk 1]
```diff
@@ -50,14 +50,14 @@ def test_bot(self):
 
         # Test help.
         bot_response = '''
-            This bot allows users to translate a sentence into
+            This bot will allow users to translate a sentence into
```
Comment (neiljp) [inline zulip_bots/zulip_bots/bots/yoda/test_yoda.py]: Either form of this reads fine to me.

Comment (derAnfaenger) [inline zulip_bots/zulip_bots/bots/yoda/test_yoda.py]: It's updating the test for the yoda bot, so `tools/test-bots` doesn't fail on yoda.

**Comment by derAnfaenger** [comment]
Once Travis finishes on this without any errors, this can be merged.

**Comment by derAnfaenger** [comment]
I'll look into this tomorrow.

**Comment by timabbott** [comment]
@abhijeetkaur fixed the other reference in https://github.com/zulip/python-zulip-api/pull/68

**Pull request closed without merging**